opentelemetry-sdk/src/opentelemetry/sdk/util/severity.py

```diff
@@ -0,0 +1,71 @@
+# Copyright The OpenTelemetry Authors
```

Comment:
I'm wondering if we should put this in the `logs` namespace? Severity seems pretty specific to only logging.

Reply:
Makes sense to me